.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,59 +1,25 @@
-<!--
-Thanks for your contribution to ManimCommunity!
-
-Before filling in the details, ensure:
-- Your local changes are up-to-date with ManimCommunity/manim
-  
-- The title of your PR gives a descriptive summary to end-users. Some examples:
-  - Fixed last animations not running to completion
-  - Added gradient support and documentation for SVG files
-  Examples of what *NOT* to do:
-  - "fixed that styling issue" - not descriptive enough
-  - "fixed issue #XYZ" - end-user needs to do further research
--->
-## Changelog / Overview
-<!-- Optional (Recommended): a detailed overview of the PR for the upcoming
-release's changelog entry. Useful for when the PR title isn't enough. 
-
-DO NOT REMOVE THE FOLLOWING CHANGELOG LINES, EVEN IF YOU DON'T USE THEM.-->
```

Comment:
Is this warning not required anymore? It won't break anything with the changelog generation right?

Reply:
I'm pretty sure I have deleted the section in #1850 and stuff seemed to work fine nonetheless.